webapp/utils/aws/ecs.py

```diff
@@ -35,30 +36,31 @@ class ECSClient:
     def client(self) -> "ECSClientType":
         return boto3.client("ecs")
 
-    @property
-    def task_family_revision(self) -> str:
-        return self.task_definition.split("/")[-1]
-
     @property
     def task_family(self) -> str:
         return re.sub(":.*", "", self.task_family_revision)
 
-    def execute_review_run(self) -> dict | None:
+    @property
+    def task_family_revision(self) -> str:
+        return self.task_definition.split("/")[-1]
+
+    def execute_review_run(self) -> str | None:
```

Comment:
Non-blocking: just curious how the type hint for this method shifted without any code changes?

Reply:
Ah! That is because the `execute_review_run` and `execute_final_run` methods rely on the submethod `run`; that submethod is where changes took place (removed `parse_run_details` to return just the task ARN).

Reply:
Got it, thanks!